spring-cloud-task-core/src/test/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDaoTests.java

```diff
@@ -166,6 +166,72 @@ public void testFindAllDefaultSort()  {
 		assertEquals("FOO3", taskExecution.getTaskName());
 	}
 
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNames()  {
+		super.getLatestTaskExecutionsByTaskNamesWithSingleTaskName();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNamesWithNullParameter()  {
+		super.getLatestTaskExecutionsByTaskNamesWithNullParameter();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNamesWithEmptyArrayParameter()  {
+		super.getLatestTaskExecutionsByTaskNamesWithEmptyArrayParameter();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNamesWithArrayParametersContainingNullAndEmptyValues()  {
+		super.getLatestTaskExecutionsByTaskNamesWithArrayParametersContainingNullAndEmptyValues();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNamesWithMultipleTaskNames()  {
+		super.getLatestTaskExecutionsByTaskNamesWithMultipleTaskNames();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionsByTaskNamesWithIdenticalTaskExecutions()  {
+		super.getLatestTaskExecutionsByTaskNamesWithIdenticalTaskExecutions();
+	}
+
+	@Test
+	@DirtiesContext
+	public void tesGetLatestTaskExecutionForTaskNameWithNullParameter()  {
+		super.getLatestTaskExecutionForTaskNameWithNullParameter();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionForTaskNameWithEmptyStringParameter()  {
+		super.getLatestTaskExecutionForTaskNameWithEmptyStringParameter();;
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionForNonExistingTaskName()  {
+		super.getLatestTaskExecutionForNonExistingTaskName();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionForExistingTaskName()  {
+		super.getLatestTaskExecutionForExistingTaskName();
+	}
+
+	@Test
+	@DirtiesContext
+	public void testGetLatestTaskExecutionForTaskNameWithIdenticalTaskExecutions()  {
+		super.getLatestTaskExecutionForTaskNameWithIdenticalTaskExecutions();
+	}
+
```

Comment:
For this block, see previous comments about adding `@Test` at the super class level.